src/providers/swift.rs

```diff
@@ -0,0 +1,147 @@
+use super::Provider;
+use crate::nixpacks::{
+    app::App,
+    environment::Environment,
+    nix::pkg::Pkg,
+    phase::{BuildPhase, InstallPhase, SetupPhase, StartPhase},
+};
+use anyhow::Result;
+use indoc::formatdoc;
+
+static LATEST_SWIFT: &str = "5.6.1";
```

Comment:
I believe this should be named DEFAULT_SWIFT_VERSION.

Reply:
done